proto/sbom/sbom.proto

```diff
@@ -29,7 +30,8 @@ message SBOMEntity {
   string                             hash               = 9; // Hash of the SBOM
   oneof sbom {
     cyclonedx.v1_4.Bom cyclonedx = 10; // only cyclonedx will be supported initially but putting it optional will allow us to move to another format later
```

Comment:
I’m wondering if it would make sense to rename that field `cyclonedx_1_4` for homogeneity with the new field.

```suggestion
    cyclonedx.v1_4.Bom cyclonedx_1_4 = 10; // only cyclonedx will be supported initially but putting it optional will allow us to move to another format later
```

It would be a non source compatible change that will require to update:
* the `datadog-agent` code [here](https://github.com/DataDog/datadog-agent/blob/67c9f3c029edf864909b5ecd9a6b5c8111a899bd/pkg/collector/corechecks/sbom/processor.go#L213)
* the `dd-source` code [here](https://github.com/DataDog/dd-source/blob/c9ccabf5c5d6b663ad1853ae0bb0d550e17c91ec/domains/cloud-security-platform/asm-vm/apps/code-workload-runner-sca/worker/worker.go#L274) and [here](https://github.com/DataDog/dd-source/blob/c9ccabf5c5d6b663ad1853ae0bb0d550e17c91ec/domains/static-analysis/apps/apis/static-analysis-sca-api/internal/services/evp_sbom_impl.go#L58)
* full list [here](https://github.com/search?q=org%3ADataDog+SBOMEntity_CycloneDX&type=code)

when bumping the version of `agent-payload` in those repositories.
But as long as the ID (`10`) remains unchanged, it has no impact on the on-wire message.

Reply:
Agreed. Can we plan that in another PR ?